### PR TITLE
add pyrad tool

### PR DESCRIPTION
**Pull request by cmonjeau**
This is the pyrad tool for assembly and analysis of RADseq data sets.

### CODE REVIEW

File: recipes/pyrad/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package:
+  name: pyrad
+  version: '3.0.64'
+
+source:
+  fn: 3.0.64.tar.gz
+  url: https://github.com/dereneaton/pyrad/archive/3.0.64.tar.gz
```
Comment (bgruening) [inline recipes/pyrad/meta.yaml]: sha256 checksum would be nice

File: recipes/pyrad/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - numpy
+    - scipy
+    - six
+    - muscle
+    - vsearch
+
+
+test:
+  imports:
+    - pyrad
```
Comment (bgruening) [inline recipes/pyrad/meta.yaml]: A few more imports would be nice to see if potential modules are installed as well. For example:

import pyrad.packet
import pyrad.client
import pyrad.dictionary

File: recipes/pyrad/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+package:
+  name: pyrad
+  version: '3.0.64'
+
+source:
+  fn: 3.0.64.tar.gz
+  url: https://github.com/dereneaton/pyrad/archive/3.0.64.tar.gz
+
+build:
+  number: 0
```
Comment (bgruening) [inline recipes/pyrad/meta.yaml]: pyrad is python 2 only, right?
Than please add `skip: True # [not py27]`

**Comment by bgruening** [comment]
Let's see if the tests are passing after your modifications :)

**Comment by bgruening** [comment]
:+1: from my side. Can you please merge master into this branch.
Thanks!

**Comment by johanneskoester** [comment]
Thanks!

**Pull request merged**